targets/cpp-ue4/Plugins/PlayFab/Source/PlayFab/Public/Core/PlayFabJsonHelpers.h

```diff
@@ -1,13 +1,16 @@
 #pragma once
 
+class FJsonObject;
+class FJsonValue;
+
 namespace PlayFab
 {
 	class FPlayFabJsonHelpers
 	{
 	public:
 
 		// Returns the array named Key or nullptr if it is missing or the wrong type
-		static const TArray< TSharedPtr<FJsonValue> >& ReadArray(TSharedPtr<class FJsonObject> Item, const FString& Key);
+		static const TArray< TSharedPtr<class FJsonValue> >& ReadArray(TSharedPtr<class FJsonObject> Item, const FString& Key);
```

Comment:
What does this do?

Reply:
I had to make changes as UE4 moves away from its monolithic way of loading classes. I am just defining the class without the headers here, and then include the headers in CPP to avoid addition overhead.

Reply:
Optimization.  Got it.  Sounds good.